src/main/java/org/veupathdb/service/eda/compute/plugins/correlationassaymetadata/CorrelationAssayMetadataPlugin.java

```diff
@@ -85,6 +86,19 @@ protected void execute() {
     String method = computeConfig.getCorrelationMethod().getValue();
     CollectionSpec collectionVariable = computeConfig.getCollectionVariable();
     String entityId = collectionVariable.getEntityId();
+    FeaturePrefilterThresholds featureFilterThresholds = computeConfig.getPrefilterThresholds();
+    String proportionNonZeroThresholdRParam = 
+      featureFilterThresholds != null &&
+      featureFilterThresholds.getProportionNonZero() != null ? 
+        ",proportionNonZeroThreshold=" + featureFilterThresholds.getProportionNonZero() : "";
```

Comment:
why not set some defaults? Even if the default is 0. Or did that get added to the microbiomeComputations side?

Reply:
Yes they're in the R